pyfolio/perf_attrib.py

```diff
@@ -361,20 +361,25 @@ def plot_returns(perf_attrib_data, cost=None, ax=None):
     total_returns_label = 'Total returns'
 
     if cost is not None:
-        returns = returns - cost
+        cumulative_returns_less_costs = _cumulative_returns_less_costs(
+            returns,
+            cost
+        )
         total_returns_label += ' (adjusted)'
 
     specific_returns = perf_attrib_data['specific_returns']
     common_returns = perf_attrib_data['common_returns']
 
-    ax.plot(ep.cum_returns(returns), color='b', label=total_returns_label)
+    ax.plot(cumulative_returns_less_costs, color='b',
```

Comment:
I think this means we're only plotting returns now if `cost` isn't None?

Reply:
oh yup, fixed that